code/datums/components/tattoo.dm

```diff
@@ -0,0 +1,60 @@
+/**
+ * # engraved component!
```

Comment:
🍝

Reply:
aaaaaaaaaaaaaaaaaaaaaaa one # will make it TOO BIG!